### DISCUSSION TITLE
Default Language/Value for Translations

### DISCUSSION THREAD

**Discussion by st11x** [directus/directus#13457] [Archive]
Hi,

I am trying to add translations to an existing collection. I understand that I'll have to duplicate the existing fields to the new collection. 

First when creating the translations field, what does the default value mean? Once created, you cannot change that value again as it is no longer exposed in the UI. 

I had set it to en-US thinking it is going to set the default translation to that but it didn't. 

![image](https://user-images.githubusercontent.com/7398499/169578212-75d65307-f60d-4fa3-a807-211bad71e8d2.png)

For some reasons, the default is set to de-DE; likely the first in the list got picked.

![image](https://user-images.githubusercontent.com/7398499/169578517-d0d229d4-794b-4238-8718-493bd0010b7b.png)

My project is set to en-US,

![image](https://user-images.githubusercontent.com/7398499/169578669-e5f70574-471d-4f49-82d5-65a27abcd65b.png)

and the user language is set to the default. 

![image](https://user-images.githubusercontent.com/7398499/169578802-bba8839c-4e47-442e-8936-e6e542b82a1a.png)

If I changed the user preference to say French, then the field would also change to French, but default didn't do anything. 

I did not check this option even for the French to work.

![image](https://user-images.githubusercontent.com/7398499/169579159-157c2c10-afdb-4022-8c5c-0b03864e2305.png)

thanks
  Matt

**Comment by azrikahar** [comment]
> First when creating the translations field, what does the default value mean? Once created, you cannot change that value again as it is no longer exposed in the UI.

The default value is technically the same as any other fields in that it's to set the initial value even if we were to leave it blank on creation. However as you correctly pointed out, alias fields in general (which includes translations, M2M, O2M etc) does not show the default value input after it is created:

https://github.com/directus/directus/blob/a7534c1211e7c0dfdca98ac4caced0a89561c43e/app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-schema.vue#L73-L74

So perhaps it should just be removed from the fields simple mode UI for alias fields? cc @rijkvanzanten 

> For some reasons, the default is set to de-DE; likely the first in the list got picked.

Yes this is the current behavior (first in the list).

> If I changed the user preference to say French, then the field would also change to French, but default didn't do anything.
>
> I did not check this option even for the French to work.

By "change to French", you mean when entering translations right? 

Regarding the not checking "Use current user language", I believe that is for the _display_ (such as viewing them on a table layout), not the interface (which is when you are entering translations), so that does sounds like it is currently working as intended.

**Reply by st11x** [reply]
By "change to French", I meant changing the language in the User Preferences. So that field influences which content translation is shown by default which is great, so a user that works on a particular translation is shown that right-away.  

I was thinking as in the screenshots that if you had set the default language for the project, then all users will see that language under translations first (it doesn't work like that). However, if the user has a preferred language set, then it uses that (this works).  The "Use Current User Language" doesn't seem to do anything.
